packages/demo/src/main.tsx

```diff
@@ -6,3 +6,9 @@ const rootNode = createRootElement()
 document.body.appendChild(rootNode)
 
 createRoot(rootNode).render(<App />)
+
+// This error replaces another error which message is harder to understand and impossible to avoid util the issue https://github.com/storyblok/field-plugin/issues/107 has been resolved.
+// eslint-disable-next-line functional/no-throw-statement
+throw new Error(
```

Comment:
Doesn't really belong to the topic in this pull request, but I figured we should add it before we forget it. (The demo app was created before we added this to the templates)